src/twig.core.js

```diff
@@ -166,20 +181,20 @@ var Twig = (function (Twig) {
      *        a new context.
      */
     Twig.Context = function(context) {
+        context = context || {};
+
         // initializing Twig.Context over an existing context 
         // should return that context.
-        if (context instanceof Twig.Context) {
-            return context;
+        if (!context._twig_merge) {
+            for (var key in Twig.Context.Utils) {
+                context[key] = Twig.Context.Utils[key].bind(context);
```

Comment:
Hi @justjohn, I'm not sure it's okay to copy methods from `Twig.Context.Utils` into the context directly.
Now `{{ _context|keys }}` returns '_twig_create,_twig_merge'. Of course we can hack that filter (as well as others) but I think there should be a nicer way.